File: MT.Model/Dictionary/PartyPosition.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace MT.Model.Dictionary
{
    #region PartyPosition:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a PartyPosition:EntityBase.
    /// </summary>
    public class PartyPosition : EntityBase
    {
        [Key]
        public Guid PartyPositionID { get; set; }
        public string PartyPositionCode { get; set; }
        public string PartyPositionName { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
  
    }
    #endregion
}

File: BECore/app/model/PartyPosition.js
/// <summary>
/// Model EXTJS PartyPosition.
/// </summary>
Ext.define('QLDT.model.PartyPosition', {
    extend: 'Ext.data.Model',
    idProperty: 'PartyPositionID',
    idType: 'Guid',

    code: 'PartyPositionCode',
    name: 'PartyPositionName',
    fields: [
		{ name: "PartyPositionID", type: "string" },
		{ name: "PartyPositionCode", type: "string" },
		{ name: "PartyPositionName", type: "string" },
		{ name: "Description", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

